### DISCUSSION TITLE
getDMMF not returning @relation fields argument

### DISCUSSION THREAD

**Discussion by gfpacheco** [prisma/prisma#2096] [Q&A] [answered]
I have a schema like this:

```
model Post {
  id       Int    @default(autoincrement()) @id
  author   User   @relation(fields: [authorId], references: [id])
  authorId Int
}
```

And I'm using `getDMMF` to get the schema information so I can generate code for it.

But it doesn't tell me the `fields` argument for the `@relation` directive. This is all I get for the `author` field:

```
    {
      "name": "author",
      "kind": "object",
      "dbNames": [],
      "isList": false,
      "isRequired": true,
      "isUnique": false,
      "isId": false,
      "type": "User",
      "relationName": "PostToUser",
      "relationToFields": [
        "id"
      ],
      "relationOnDelete": "NONE",
      "isGenerated": false,
      "isUpdatedAt": false
    },
```

Is there any way to get that information?

**Comment by timsuchanek** [comment]
Thanks for asking  @gfpacheco !
As the `fields` argument of the `@relation` directive is fairly new, we didn't add it to the DMMF yet. I will ping you here, once we have added it.

**Comment by timsuchanek** [comment, marked as answer]
This is fixed in the latest alpha, please try out `@prisma/cli@alpha`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
